server/index.js

```diff
@@ -0,0 +1,14 @@
+//  import statements
+const express = require('express');
+// const path = require('path');
+
+const filePath = '/Users/robertlopez/Desktop/Bluefin-Mortgage-Service/public';
```

Comment:
you should probably use the path module so that other teammates can use it if they want to download the repo and run the server on their computer

Reply:
nice suggestion! Will definitely update!